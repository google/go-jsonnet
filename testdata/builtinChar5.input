// too big
std.char(1114112)
