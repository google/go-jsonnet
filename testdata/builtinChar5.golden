RUNTIME ERROR: Invalid unicode codepoint, got 1.114112e+06
-------------------------------------------------
	<builtin>	builtin function <char>

-------------------------------------------------
	During evaluation	


